### ISSUE TITLE
Documentation for v5 missing

### ISSUE THREAD

**Issue by attekemppila** [eslint-stylistic/eslint-stylistic#1239]
### Validations

- [x] Follow our [Code of Conduct](https://github.com/eslint-stylistic/eslint-stylistic/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [Contributing Guide](https://eslint.style/contribute/guide).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

### Describe the bug

https://eslint.style/ "current" version is v6.0.0-beta5. Documentation for older v4 is also available: https://v4.eslint.style/.

But the actual current v5 documentation is not available on the website.

I ran into this when a rule that I was looking at differed in v5 and v6-beta.

### Reproduction

https://eslint.style/

### System Info

```shell
-
```

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests
